test/Tester.cpp

```diff
@@ -0,0 +1,7 @@
+#include <gtest/gtest.h>
+
+int main(int argc, char **argv)
+{
+    ::testing::InitGoogleTest(&argc, argv);
+    return RUN_ALL_TESTS();
```

Comment:
A unit tesztekkel kapcsolatban keresem a workflowt, de nem találom.

Ezt pótolni kéne, mivel még az előző feladat része volt (#6) és továbbra is futnia kellene.